src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -1344,6 +1350,38 @@ private void servedC100Order(CaseData caseData, List<Element<OrderDetails>> orde
         );
     }
 
+    private List<Element<ServedParties>> getServedParties(CaseData caseData) {
+        List<Element<ServedParties>> servedParties = new ArrayList<>();
+        if (caseData.getManageOrders()
+            .getRecipientsOptions() != null) {
+            servedParties = dynamicMultiSelectListService
+                .getServedPartyDetailsFromDynamicSelectList(caseData
+                                                                .getManageOrders()
+                                                                .getRecipientsOptions());
+        } else if (caseData.getManageOrders()
+            .getRecipientsOptionsOnlyC47a() != null) {
+            servedParties = dynamicMultiSelectListService
+                .getServedPartyDetailsFromDynamicSelectList(caseData.getManageOrders()
+                                                         .getRecipientsOptionsOnlyC47a());
+        }
+        if (caseData.getManageOrders().getChildOption() != null) {
+            servedParties.addAll(dynamicMultiSelectListService
+                                     .getServedPartyDetailsFromDynamicSelectList(caseData.getManageOrders()
+                                                                                     .getChildOption()));
+        }
```

Comment:
For FL401, do we bother about childs involved while drawing order? If not, we can put another & for case type please.